linbox/matrix/sparsematrix/Makefile.am

```diff
@@ -38,15 +40,20 @@ pkgincludesub_HEADERS =         \
 	sparse-generic.inl \
 	sparse-associative-vector.h      \
 	sparse-associative-vector.inl    \
-        sparse-parallel-vector.h         \
-        sparse-parallel-vector.inl       \
-        sparse-sequence-vector.h         \
+	sparse-parallel-vector.h         \
+	sparse-parallel-vector.inl       \
+	sparse-sequence-vector.h         \
 	triples-coord.h  \
 	read-write-sparse.h \
 	read-write-sparse.inl \
+<<<<<<< HEAD
```

Comment:
You have commited merge data (<<<HEAD, ====, etc). Please remove them.

Reply:
Oops.  Now fixed.  Thnx.